### ISSUE TITLE
UTD-MHAD window length, preprocessing code

### ISSUE THREAD

**Issue by shopeth** [arturjordao/WearableSensorData#1]
Thank you so much for sharing the processed data and code of different methods. These are valuable for the research in this domain.

One quick question is that it seems on UTD-MHAD dataset a 1-sec window is used while for other datasets the standard 5-sec window is used. What was the reason for this?
Also, I wonder if it is possible to share the preprocessing code on each dataset too (from raw recording to the npz windowed data)?

Thank you!

**Comment by arturjordao** [comment]
Dear @shopeth , 

The activities of UTD-MHAD have a duration of 2-3 seconds (on average). Thus, it is not possible to employ a 5-seconds window. Therefore, to this dataset, we have used a one-second window.

I have attached the codes to convert and generate the data from each dataset. Unfortunately, I'm not sure if these codes are the final version used in the paper, thus, they can contain some bugs...
Basically, it is necessary to convert the dataset into files such as:
"activity1_subject1_Id1.txt",
"activity2_subject1_Id2.txt",
"activity1_subject2_Id3.txt",
"activity4_subject2_Id4.txt",
"activity3_subjectN_Id5.txt",
"activity2_subjectN_Id6.txt",
Then, run the desired protocol (e.g., data_generator_loso.py), which will yield the npz file.
[ExtraCodes.zip](https://github.com/arturjordao/WearableSensorData/files/3056477/ExtraCodes.zip)
